api/openapi/bootstrap.yml

```diff
@@ -533,3 +501,31 @@ components:
         application/json:
           schema:
             $ref: "./schemas/HealthInfo.yml"
+
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+      description: |
+        * Users access: "Authorization: Bearer <user_token>"
+
+    thingAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: uuid
+      description: |
+        * Things access: "Authorization: Thing <external_key>"
+
+    thingEncAuth:
```

Comment:
Probably `bootstrapAuth` is a better naming.

Reply:
Probably better yes